google/cloud/bigtable/internal/instance_admin.h

```diff
@@ -109,38 +111,89 @@ class InstanceAdmin {
    * `bigtable::InstanceAdmin` class, but do not raise exceptions on errors,
    * instead they return the error on the status parameter.
    */
-  std::vector<::google::bigtable::admin::v2::Instance> ListInstances(
+  std::vector<google::bigtable::admin::v2::Instance> ListInstances(
       grpc::Status& status);
 
-  ::google::bigtable::admin::v2::Instance GetInstance(
+  google::bigtable::admin::v2::Instance GetInstance(
       std::string const& instance_id, grpc::Status& status);
 
+  /**
+   * Make an asynchronous request to get the instance.
+   *
+   * @param instance_id the id of the instance in the project that needs to be
+   *     Retrieve.
+   * @param cq the completion queue that will execute the asynchronous calls,
+   *     the application must ensure that one or more threads are blocked on
+   *     `cq.Run()`.
+   * @param callback a functor to be called when the operation completes. It
+   *     must satisfy (using C++17 types):
+   *     static_assert(std::is_invocable_v<
+   *         Functor, google::bigtable::v2::MutateRowResponse&,
+   *         grpc::Status const&>);
+   *
+   * @tparam Functor the type of the callback.
+   *
+   * @return A handle to the asynchronous operation, applications can request
```

Comment:
Actually this returns `void`...  I neglected to fix the comments in the other functions too... Please remove this comment about `@return`. I will fix the other functions. And I need to think if these operations should be cancelable.